zerver/openapi/zulip.yaml

```diff
@@ -8016,6 +8040,205 @@ paths:
+          schema:
+            type: boolean
+          example: true
+        - name: fluid_layout_width
+          in: query
+          description: |
+            Use maximum available screen width for display in large screens.
+          schema:
+            type: boolean
+          example: true
+        - name: high_contrast_mode
+          in: query
+          description: |
+            It is an experimental setting, that may be available in future.
+          schema:
+            type: boolean
+          example: true
+        - name: color_scheme
+          in: query
+          description: |
+            Color scheme for display.
+
+            * 1 - Automatic
+            * 2 - Night mode
+            * 3 - Day mode
+          content:
+            application/json:
+              schema:
+                type: integer
+                enum:
+                  - 1
+                  - 2
+                  - 3
+              example: 1
+        - name: translate_emoticons
+          in: query
+          description: |
+            Allow translation of emoticons to corresponding emojis.
+          schema:
+            type: boolean
+          example: true
+        - name: default_language
+          in: query
+          description: |
+            Set default language. It should be the language code of new language to be set.
+
+            for example - `"en" for English` or `"de" for Deutsch`
+          content:
+            application/json:
+              schema:
+                type: string
+              example: "en"
+        - name: left_side_userlist
+          in: query
+          description: |
+            Show user list on left sidebar in narrow windows.
+          schema:
+            type: boolean
+          example: true
+        - name: emojiset
+          in: query
+          description: |
+            Emoji set to be used for display.
+
+            * "google" - Google modern
+            * "google-blob" - Google classic
+            * "twitter" - Twitter
+            * "text" - Plain text
+          schema:
+            type: string
+          example: '"google"'
+        - name: demote_inactive_streams
+          in: query
+          description: |
+            Set level of filtering for demoting inactive streams.
+
+            * 1 - Automatic
+            * 2 - Always
+            * 3 - Never
+          content:
+            application/json:
+              schema:
+                type: integer
+                enum:
+                  - 1
+                  - 2
+                  - 3
+              example: 1
+        - name: timezone
+          in: query
+          description: |
+            Set default timezone.
+            It could be any valid timezone from [Olson tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+          content:
+            application/json:
+              schema:
+                type: string
+              example: "Asia/Kolkata"
```

Comment:
For all of these settings, can you add a link in the description to the appropriate part of the `/help/` documentation covering the setting in question?  This sort of cross-linking is really important to making our API documentation useful.

Reply:
I made this changes although I am not 100% sure if the sentence I have added is correct. This is the [commit](https://github.com/zulip/zulip/pull/17033/commits/134aa4eaeafc11dfe4eb9a7207c625e030775cfd) that adds link for this /display api documentation in corresponding help docs. Please have a look.